### PR TITLE
[bernabe_chavajay]my google version BC

### PR DESCRIPTION
**Pull request by bchavajay**
my google version 1.0

### CODE REVIEW

File: bernabe_chavajay/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <div class="navbar">
+        <button class="settings">Google</button>
+
+        <a href="https://mail.google.com">Gmail</a>
+
+        <a href="https://www.google.com.gt/imghp?hl=es&authuser=0&ogbl">Imagenes</a>
+    </div>
+
+    <div class="logo">
+        <img src="logo.png" alt="logo-google"
```
Comment (kenri) [inline bernabe_chavajay/index.html]: Please use the image from the url and not include the image on the repository

**Review by kenri — CHANGES_REQUESTED** [review]

File: bernabe_chavajay/index.html (hunk 1 above)
Comment (bchavajay) [reply bernabe_chavajay/index.html]: Got it

Comment (kenri) [reply bernabe_chavajay/index.html]: Waiting for this change

Comment (bchavajay) [reply bernabe_chavajay/index.html]: I did the change

**Review by kenri — APPROVED** [review]

**Pull request merged**